devenv/docker/blocks/intentapi/README.md

```diff
@@ -0,0 +1,31 @@
+# Intent API devenv
+
+This devenv stands up a `kube-apiserver` and a single-host `etcd` for local development / testing.
+
+## kubeapi and etcd versions
+
+You can switch to different kubeapi and etcd versions by changing the values in `.env`. Make sure that the images for those versions exist and can be pulled.
+
+## Preparing the environment
+
+In order to run and use `kube-apiserver` we need to setup PKI keys and certificates. This can be done with `make` - ensure that you do that before you boot up the environment, or re-create the environment after doing that in order for the changes to take effect:
+
+```sh
+# Install cfsll (https://github.com/cloudflare/cfssl):
+$ go get github.com/cloudflare/cfssl/cmd/cfssl
+
+# Or, with Homebrew:
+$ brew install cfssl
+
+# Generate certs and kubeconfig:
+$ make
+```
+
+Once the certs are created, you can run the environment as a usual devenv Docker block:
+```sh
+# Assuming that you're in repo's root:
+$ make devenv sources=intentapi
+
+# Test that the environment is working correctly:
+$ kubectl --kubeconfig=devenv/docker/blocks/intentapi/admin.kubeconfig api-resources
```

Comment:
@papagian I _think_ this might be because I've accidentally committed my kubeconfig. Could you try recreating the setup from scratch and test if it works?
```sh
$ cd devenv/docker/blocks/intentapi
$ make clean
$ make all
$ cd ../../../../

# Stop and delete existing containers

$ make devenv sources=intentapi
$ kubectl --kubeconfig=devenv/docker/blocks/intentapi/admin.kubeconfig api-resources
```

Reply:
👍  it works now!